assets/js/stork-search.js

```diff
@@ -0,0 +1,97 @@
+/*
+ *
+ * Copyright 2021 Adobe
+ * All Rights Reserved.
+ *
+ * NOTICE: Adobe permits you to use, modify, and distribute this file in
+ * accordance with the terms of the Adobe license agreement accompanying
+ * it. If you have received this file from a source other than Adobe,
+ * then your use, modification, or distribution of it requires the prior
+ * written permission of Adobe.
+ *
+ */
+const initPromise = stork.initialize("{{ .Site.BaseURL }}/stork/stork.wasm");
+const downloadPromise = stork.downloadIndex("site-search-index", "{{ .Site.BaseURL }}/stork/site-index.st");
```

Comment:
should we add `{ forceOverwrite: true }` ?

Reply:
yep, good idea. added.